dnf5/config/usr/share/dnf5/aliases.d/compatibility.conf

```diff
@@ -145,6 +145,13 @@ descr = "Alias for 'remove'"
 group_id = 'commands-compatibility-aliases'
 complete = false
 
+['rq']
+type = 'command'
+attached_command = 'repoquery'
+descr = "Alias for 'repoquery'"
+group_id = 'commands-compatibility-aliases'
+complete = true
```

Comment:
We generally don't have command auto-completion turned on for these shortcuts, can you set
```
complete = false
```

Reply:
Sure, done.  Would a follow-up and/or separate patch to change the `ls` alias be welcome as well?  Then there would be no 2-character aliases which are completed.